engine/graphics/src/vulkan/graphics_vulkan_context.cpp

```diff
@@ -61,7 +61,7 @@ namespace dmGraphics
     {
         if (!IsSupported(supported_extensions, extension))
         {
-            dmLogWarning("%s was not supported", extension);
+            dmLogDebug("%s was not supported", extension);
```

Comment:
I think people will be annoyed by this, and we only add extensions if they are supported. Meaning, they are not required extensions to do rendering.